onnxruntime/core/optimizer/graph_transformer_utils.cc

```diff
@@ -45,15 +48,20 @@ std::vector<TransformerProviderSet> GenerateTransformers(const TransformerLevel&
                                                          const std::vector<std::string>* transformers_to_enable) {
   std::vector<TransformerProviderSet> transformers;
   switch (level) {
-    case TransformerLevel::Level1: {
-      std::vector<std::string> l1_execution_providers = {};
-      transformers.emplace_back(std::make_unique<UnsqueezeElimination>(), l1_execution_providers);
-    } break;
+    case TransformerLevel::Level1: /*{
+      //std::vector<std::string> l1_execution_providers = {};
+      //transformers.emplace_back(std::make_unique<UnsqueezeElimination>(), l1_execution_providers);
+    }*/ break;
 
     case TransformerLevel::Level2: {
       std::vector<std::string> l2_execution_providers = {onnxruntime::kCpuExecutionProvider};
-      transformers.emplace_back(std::make_unique<ConvAddFusion>(), l2_execution_providers);
-      transformers.emplace_back(std::make_unique<ConvMulFusion>(), l2_execution_providers);
+      std::vector<std::string> l2_global = {};
+      //transformers.emplace_back(std::make_unique<ConvActivationFusion>(), l2_execution_providers);
+      transformers.emplace_back(std::make_unique<ConvBNFusion>(), l2_execution_providers);
+      transformers.emplace_back(std::make_unique<GemmActivationFusion>(), l2_execution_providers);
+      transformers.emplace_back(std::make_unique<MatMulAddFusion>(), l2_execution_providers);
+      transformers.emplace_back(std::make_unique<ConvAddFusion>(), l2_global);
+      transformers.emplace_back(std::make_unique<ConvMulFusion>(), l2_global);
```

Comment:
should ConvBN, ConvAdd, ConvMul be L1? The semantic is preserved in these ops fusion and they could be applied before partition.

Reply:
They are not supported by training mode... That's why it was decided to add them as L2 transformers... L1 transformers are the once which are supported for all execution providers and all modes (inference and training)

Reply:
should ConvBNFusion be l2_global?

Reply:
Level1

Reply:
If so, how can converter use these optimizer: ConvBN, ConvAdd and ConvMul?

Reply:
This is a trade-off that we are making at this moment to keep things simple... and this is out of scope for this PR... We can discuss this offline as to whether we want to add another flag for training session or other ways... I will set a meeting for this...

Reply:
Yes, we can discuss it in the meeting. Thanks!